installing/installing-fips.adoc

```diff
@@ -5,11 +5,16 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Starting with version 4.3, you can install an {product-title} cluster that uses FIPS Validated / Modules in Process cryptographic libraries.
+You can install an {product-title} cluster that uses FIPS Validated / Modules in Process cryptographic libraries.
 
-For the {op-system-first} machines in your cluster, this change is applied when the machines are deployed based on the status of an option in the `install-config.yaml` file, which governs the cluster options that a user can change during cluster deployment. With {op-system-base-full} machines, you must enable FIPS mode when you install the operating system on the machines that you plan to use as worker machines. These configuration methods ensure that your cluster meet the requirements of a FIPS compliance audit: only FIPS Validated / Modules in Process cryptography packages are enabled before the initial system boot.
+For the {op-system-first} machines in your cluster, this change is applied when the machines are deployed based on the status of an option in the `install-config.yaml` file, which governs the cluster options that a user can change during cluster deployment. With {op-system-base-full} machines, you must enable FIPS mode when you install the operating system on the machines that you plan to use as worker machines. These configuration methods ensure that your cluster meets the requirements of a FIPS compliance audit, indicating that only FIPS Validated / Modules in Process cryptography packages are enabled before the initial system boot.
 
-Because FIPS must be enabled before the operating system that your cluster uses boots for the first time, you cannot enable FIPS after you deploy a cluster.
+Because FIPS must be enabled before the cluster's operating system boots for the first time, you cannot enable FIPS after you deploy a cluster.
+
+[IMPORTANT]
+====
+For LUKS encryption, you must change the default LUKS cipher to be FIPS-approved if you want to enable default FIPS compliance on {op-system} machines.
+====
```

Comment:
@marrusl Not sure how close this is to what you want to convey in the [FIPS crypto](https://docs.openshift.com/container-platform/4.6/installing/installing-fips.html) section, but wanted to have some place to start.

Reply:
The docs for enabling LUKS now include the cipher change.  It might be better to drop this warning now, since the advice might be more confusing than helpful.

Reply:
@bgilbert Does it make sense to include this note or similar wording here? This would be in addition to your configs in https://github.com/openshift/openshift-docs/pull/28637 and https://github.com/openshift/openshift-docs/pull/29130.

Reply:
Perfect, that's good to know, @bgilbert. I will close this PR. Thanks!